packages/amplify_api/example/.gitignore

```diff
@@ -57,7 +57,8 @@ node_modules/
 aws-exports.js
 awsconfiguration.json
 amplifyconfiguration.json
```

Comment:
*nit:* You could probably remove `amplifyconfiguration.json`. I believe that is what the CLI does for flutter projects now. I think `.dart` is missing and `.json` is present because the CLI use to add `.json` even for flutter projects, but that has been updated.